include/IVideoDriver.h

```diff
@@ -182,7 +183,7 @@ namespace video
 
 		virtual void removeMultisampleTexture(IMultisampleTexture* tex) =0;
 
-		virtual void removeTextureBufferObject(ITextureBufferObject* tbo) =0;
+		virtual void removeTextureBufferObject(IGPUBufferView* tbo) =0;
```

Comment:
why did we need `IDriver::remove*` functions in the first place and just not be able to `drop()` stuff?

Reply:
because they're cached in cnulldriver and remove*() removes the object from cache

Reply:
As per our discord conversation, I do not believe there should ba a cache of already created objects... especially if they are actually shareable.

There's probably no reason for this cache to exist.
Or other caches.

Reply:
resolved